tests/utils.ts

```diff
@@ -17,6 +17,69 @@ export enum TableTypes {
   AllTypes,
 }
 
+type TablePlotNames =
+  | 'all_types'
+  | 'simple_plot'
+  | 'simple_table'
+  | 'simple_table_header_group'
+  | 'simple_table_header_group_hide'
+  | 'double_and_string'
+  | 'ordered_int_and_offset'
+  | 'trig_table'
+  | 'trig_figure'
+  | 'multiselect_null'
+  | 'multiselect_empty'
+  | 'multiselect_bool'
+  | 'multiselect_datetime'
+  | 'multiselect_char'
+  | 'multiselect_number'
+  | 'multiselect_string';
+
+/**
+ * Opens a table loaded from application mode and returns the grid location
+ * @param page
+ * @param type Either 'table' or 'plot'
+ * @param name Name of the table or plot
+ */
+export async function openTableOrPlot(
```

Comment:
I'd rename this function to just `openObject`, and that takes the type option (`table` or `plot`).